Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLAutoProcess.py

```diff
@@ -328,6 +328,19 @@ def PyExec(self):
             else:
                 self.log().information('Skipping empty token run.')
 
+        for output in outputs:
+            ConvertToPointData(InputWorkspace=output,
+                               OutputWorkspace=output)
+        if len(outputs) > 1:
```

Comment:
Stitching should happen only if the output type is I(Q), so one-dimensional.

Reply:
I am now checking that OutputType is I(Q).